.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+default_language_version:
+  python: python3
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.4.0
+  hooks:
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-toml
+  - id: trailing-whitespace
+- repo: https://github.com/psf/black
+  rev: 23.7.0
+  hooks:
+  - id: black
+    args: ['--target-version', 'py38']
```

Comment:
Since the target version is already set in pyproject.toml we don't need it here and should remove the line.

Reply:
Good catch! Thank you